include/cell/copy/global_to_register.hpp

```diff
@@ -1,83 +1,190 @@
+struct GlobalToRegMatLoader {
+    using Global = Global_;
+    using BaseTile = BaseTile_;
+    using DType = Global::DType;
+
+    DEVICE void operator()(const DType* src, BaseTile& dst);
+};
+
+template <typename Global_, typename BaseTile_>
+struct GlobalToRegMatLoader<Global_, BaseTile_, tl::Layout::RowMajor> {
+    using Global = Global_;
+    using BaseTile = BaseTile_;
+    using DType = Global::DType;
+
+    static constexpr int kStride = Global::kRowStride;
+
+    DEVICE void operator()(const DType* src, BaseTile& dst) {
+        dst(0, 0) = src[0 * kStride + 0];
+        dst(0, 1) = src[0 * kStride + 1];
+        dst(1, 0) = src[0 * kStride + 8];
+        dst(1, 1) = src[0 * kStride + 9];
+        dst(0, 2) = src[8 * kStride + 0];
+        dst(0, 3) = src[8 * kStride + 1];
+        dst(1, 2) = src[8 * kStride + 8];
+        dst(1, 3) = src[8 * kStride + 9];
+    }
+};
+
+// TODO(KuangjuX): Implement this.
+template <typename Global_, typename BaseTile_>
+struct GlobalToRegMatLoader<Global_, BaseTile_, tl::Layout::ColMajor> {
+    using Global = Global_;
+    using BaseTile = BaseTile_;
+    using DType = Global::DType;
+
+    static constexpr int kStride = Global::kColStride;
+
+    DEVICE void operator()(const DType* src, BaseTile& dst) {}
+};
+
+/**
+ * @brief Load a RegTile from Global memory to Register.
+ * @tparam Global_ Global memory tile type.
+ * @tparam Reg_ Register tile type.
+ * @tparam kRowExec_ Number of times a `RegTile` is executed along the row
+ * @tparam kColExec_ Number of times a `RegTile` is executed along the column
+ * @tparam type Global Layout type.
+ */
+template <typename Global_, typename Reg_, const int kRowExec_,
+          const int kColExec_, const tl::Layout type>
 struct GlobalToRegLoaderImpl {
+    using Global = Global_;
+    using Reg = Reg_;
     using DType = Global::type;
-    static constexpr int kHeight = Reg::kRows;
-    static constexpr int kWidth = Reg::kCols;
-    static constexpr int kSubTileSize = 16;
 
-    DEVICE void operator()(const Global& src, Reg& dst);
+    DEVICE void operator()(const DType* src, Reg& dst);
 };
 
-/// @brief Load tile from global memory to register tile in RowMajor layout.
-/// @tparam Global Global tile type.
-/// @tparam Reg Register tile type.
-template <typename Global, typename Reg>
-struct GlobalToRegLoaderImpl<Global, Reg, tl::Layout::RowMajor> {
+template <typename Global_, typename Reg_, const int kRowExec_,
+          const int kColExec_>
+struct GlobalToRegLoaderImpl<Global_, Reg_, kRowExec_, kColExec_,
+                             tl::Layout::RowMajor> {
+    using Global = Global_;
+    using Reg = Reg_;
     using DType = typename Global::DType;
-    static constexpr int kHeight = Reg::kRows;
-    static constexpr int kWidth = Reg::kCols;
-    static constexpr int kSubTileSize = 16;
+    using BaseTile = typename Reg::DType;
 
-    DEVICE void operator()(const Global& src, Reg& dst) {
+    // strides to iterate over each 16x128-bits `BaseTile`.
```

Comment:
the comments here, it is now `16x16` instead of `128bits`

Reply:
I removed some constants because they are not currently used.